desktop/app-handlers/preferences/index.js

```diff
@@ -15,7 +15,7 @@ const Settings = require( 'lib/settings' );
 function promptForRestart( title, message ) {
 	// Warn user they need to restart the app
 	dialog.showMessageBox( {
-		buttons: [ 'Ok' ],
```

Comment:
This hurt my eyes.

Side note: The `spellcheck` property on the window's web preferences cannot be overriden _after_ a `BrowserWindow` is instantiated, so we still need this prompt to restart the app for the setting to take effect.